src/yang/ietf-access-control-list.yang

```diff
@@ -436,7 +436,7 @@ module ietf-access-control-list {
            and value of this name, possibly space and special
            characters are not allowed.";
       }
-      leaf acl-type {
+      leaf type {
```

Comment:
I thought Martin's feedback was that we wanted to change acl-name to name but "type" was so generic that it was better to still keep "acl-type"!?

Reply:
Here is the e-mail from Martin:

I think 99% of the changes in this PR look good.  The one
exception is the typedef that used to be called "acl-type".  I think
it should still be called "acl-type".  "type" is too broad.  NOTE,
this is just the typedef; the leaf /access-lists/acl/type should keep
its name ("type").